### PR TITLE
Support bulk delete operations for notebooks

### PR DESCRIPTION
**Pull request by joshuali925**
### Description
- observability backend supports bulk delete in one request, update notebooks to use it
- update cypress and reduce delay time

### Issues Resolved
[List any issues this PR will resolve]

### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass, including unit test, integration test and doctest
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
  - [ ] New functionality has user manual doc added
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by ps48 — APPROVED** [review]

File: dashboards-observability/server/adaptors/notebooks/default_backend.ts [hunk 1]
```diff
@@ -224,12 +224,12 @@ export class DefaultBackend implements NotebookAdaptor {
    */
   deleteNote = async function (
     client: ILegacyScopedClusterClient,
-    noteId: string,
+    noteList: string,
```
Comment (eugenesk24) [inline dashboards-observability/server/adaptors/notebooks/default_backend.ts:227]: Is noteList supposed to be string[] ??

Comment (eugenesk24) [reply dashboards-observability/server/adaptors/notebooks/default_backend.ts:227]: Oh I see, it's converted to a string for the actual call

**Review by eugenesk24 — APPROVED** [review]
Looks good!

File: dashboards-observability/server/adaptors/notebooks/default_backend.ts (hunk 1 above)
Comment (joshuali925) [reply dashboards-observability/server/adaptors/notebooks/default_backend.ts:227]: yes the ids are passed as `id1,id2,id3` in http request, this should work as long as users don't manually construct an id with comma. I might improve it later to accept an array

**Pull request merged**